templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -435,7 +453,7 @@ Parameters:
       - General Purpose (SSD)
       - Provisioned IOPS
     ConstraintDescription: Must be 'General Purpose (SSD)' or 'Provisioned IOPS'.
-    Description: Database storage type
+    Description: Database storage type #(not used for Aurora)
```

Comment:
Do we need to comment out Aurora here until we are ready to support it?

Reply:
It is  isn't it?